bridge_adaptivity/module/models.py

```diff
@@ -2,6 +2,7 @@
 import importlib
 import inspect
 import logging
+from math import floor
```

Comment:
I would prefer to import library not function to ensure we do not override something important.